storage/storage_image.go

```diff
@@ -416,8 +416,23 @@ func (s *storageImageDestination) Commit() error {
 	return nil
 }
 
+var manifestMIMETypes = []string{
+	// TODO(runcom): we'll add OCI as part of another PR here
+	manifest.DockerV2Schema2MediaType,
+	manifest.DockerV2Schema1SignedMediaType,
+	manifest.DockerV2Schema1MediaType,
+}
```

Comment:
if supportedmanifestmimetypes is nil, we do accept everything as if `IsMultiImage` returns true and copy will fail. If we list some manifest types here we do get a manifest v2s2 from the original manifest list and we'll be able to copy.